### PR TITLE
MTM-55329 changes Svenson groupId to com.nsn.cumulocity.dependencies

### PR DESCRIPTION
**Pull request by kaczynskid-sag**
And Svenson version to include original library version.

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Review by karol-c8y — DISMISSED** [review]

**Review by kowa-c8y — DISMISSED** [review]

File: .jenkins/github/pre-merge.jenkinsfile [hunk 1]
```diff
@@ -268,7 +268,7 @@ def examples_verify() {
         dir('cumulocity-examples') {
           stage('examples verify') {
             try {
-              sh ".jenkins/scripts/mvn.sh install"
+              sh ".jenkins/scripts/mvn.sh install -Dskip.agent.package=true"
```
Comment (ghost) [inline .jenkins/github/pre-merge.jenkinsfile:271]: Out of curiosity why did you add this?

**Review by ghost — APPROVED** [review]

File: .jenkins/github/pre-merge.jenkinsfile (hunk 1 above)
Comment (kaczynskid-sag) [reply .jenkins/github/pre-merge.jenkinsfile:271]: Because at this stage we only care about compilation and running tests in examples. We don't need need to create packages, as we don't use them anywhere. So it saves some tome and resources and will no longer fail if there are i.e. issues with Docker registry that I got on some of the builds. Also, microservice package plugin is tested well enough by building agents packages, that are used in tests later.

**Review by jkot-c8y — APPROVED** [review]

**Review by karol-c8y — APPROVED** [review]
